### PR TITLE
feature: Add APC NetBotz State Sensor Support

### PR DESCRIPTION
**Pull request by Rosiak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/discovery/sensors/states/netbotz.inc.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$temp = snmpwalk_cache_multi_oid($device, 'otherStateSensorTable', array(), 'NETBOTZ410-MIB');
+$cur_oid = '.1.3.6.1.4.1.5528.100.4.2.10.1.3.';
+
+if (is_array($temp)) {
+    //Create State Index
+    $state_name = 'otherStateSensorErrorStatus';
```
Comment (murrant) [inline includes/discovery/sensors/states/netbotz.inc.php]: This state name seems odd, is it meaningful?

**Comment by laf** [comment]
I've been thinking about state support a little, the state translations are basically static but if we do need to update them then it means schema files to do so.

Would it make more sense to add them into the os yaml file?

**Comment by murrant** [comment]
@laf I was thinking of converting it to something similar to the RrdDefinition class.

**Comment by laf** [comment]
I'm fine with either way, was just hoping for something a bit tidier in the sensors code. The state translations are static at the end of the day so we probably don't need them in the DB.

**Comment by murrant** [comment]
laf ah yes, I see what you mean.

**Comment by Rosiak** [comment]
Interesting thought laf, I've been wanting to refactor the code for a while makes perfectly sense to me.

File: includes/discovery/sensors/states/netbotz.inc.php (hunk 1 above)
Comment (laf) [inline includes/discovery/sensors/states/netbotz.inc.php]: It's not used anywhere and is referenced to the mib so seems appropriate if a little obscure.

**Comment by laf** [comment]
Actually, sod moving to yaml, definitely would be worse doing that.

**Pull request merged**